app/assets/javascripts/sortable.es6

```diff
@@ -40,7 +40,6 @@
         } else {
           this.flash.set("danger", "Unable to save new order.")
         }
-        this.disable()
```

Comment:
Not sure why this doesn't work, but it raises an error in the browser. Removing it doesn't seem to change functionality.